resources/views/livewire/ckeditor.blade.php

```diff
@@ -0,0 +1,42 @@
+<x-modal>
```

Comment:
I think this is something specific for what you are needing, either way, there's should be some way to render any other Livewire component that you have